main.tf

```diff
@@ -0,0 +1,81 @@
+terraform {
+  backend "azurerm" {
+    resource_group_name = "SoftwirePilot_JosephFisher_ProjectExercise"
+    storage_account_name = "storageaccountjosfis"
+    container_name = "todoapp-container"
+    key = "8O1ytbiAe65qZQZS3Sx9pDppNbJWYWSTFfklEwcVN3p0ZFbxLn4YZ5I5vv9sejmqFdrFrTDwr39xhJMabdG87Q=="
```

Comment:
You've accidentally used your access key (which is supposed to be secret) as your "Key" in the sense of key/value store. The "key" should be something like "terraform_backend.todoapp.prod". It's just the name for this blob of data.

Now everyone in the world has access to your storage account. Plus you have an unhelpful name for your terraform state.

You should rotate your keys.